### DISCUSSION TITLE
This dust and gas cloud found in space is known as the "Horsehead ... " what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#361] [Galaxy Trivia] [answered]
**Hint:** kinda haziness

**Comment by peterbaccaro** [comment]
This dust and gas cloud found in space is known as the "Horsehead Nebula." The Horsehead Nebula is a dark nebula located in the constellation Orion and is named for its distinctive shape, which resembles the head of a horse. It is a popular target for amateur and professional astronomers and has been studied extensively in visible, infrared, and radio wavelengths.

**Comment by thuongtruong1009** [comment, marked as answer]
The dust and gas cloud found in space that is commonly known as the "Horsehead" is officially called the Horsehead Nebula. It is located in the constellation Orion and is a dark nebula, meaning that it is a cloud of dust and gas that appears dark against the background of glowing gases that surround it. The Horsehead Nebula is a popular target for amateur and professional astronomers alike due to its distinctive shape and its proximity to other notable objects in the night sky.
